packages/web-component/test/descope-wc.test.ts

```diff
@@ -131,7 +134,8 @@ describe('web-component', () => {
   });
 
   afterEach(() => {
-    document.getElementsByTagName('html')[0].innerHTML = '';
+    document.getElementsByTagName('head')[0].innerHTML = '';
```

Comment:
@nirgur We made this change as it seems that our test was contaminated from previous tests. Since all tests populate `body` and `head` it seems right to reset it this way. WDYT?

Reply:
I think it's ok
we can use `document.body` & `document.head` instead